api/src/main/resources/liquibase-update-to-2.0.xml

```diff
@@ -8810,7 +8810,7 @@
     </changeSet>
     
     <changeSet id="TRUNK-5334-rename-allergy-comment-to-comments" author="madhavkauntia">
-    	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments"/>
+    	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
```

Comment:
Is there any reason why you are changing the data type?

Reply:
I haven't changed the datatype. It's the same. When I tried building and running without specifying the columnDataType attribute, the web page gave an error specifying that that attribute needs to be provided in the <renameColumn> tag. The problem was fixed when I made the required change.